docs/0-Preparing/starting_side_projects.md

```diff
@@ -0,0 +1,93 @@
+
+**You can learn a technology on your own**
+
+- *Example*: You're reading through a book on learning language X, and applying what you learned by coding a few Project Euler problems.
+- *Why it's useful*: Most employers will hire you even if you don't know all the technologies that they use, and will expect you to learn some on the job. Here, you've established that you can do that.
+
+**You can learn a concept on your own**
+
+- *Example*: You got interested in Big Data and read some blog posts about how work can be split and parallelized to run on multiple machines.
+- *Why it's useful*: Employers sometimes have hard problems to solve and like to know that they can give you a bunch of readings on a new concept.
+
+**You can come up with project ideas**
+
+- *Example*: You noticed that the school newspaper club manually tallies up survey results into different demographics. You write a script that automates this work for them and generates pretty graphs.
+- *Why it's useful*: Employers like to see that you can take initiative.
+
+**You can complete a project on your own**
+
+- *Example*: You make a website to create and send reminders like your parent's birthday. It has user authentication, and a bunch of other thing so other people can use it.
+- *Why it's useful*: It's good to complete a project at some point, as you can learn different things at different stages of a project.
+
+As you can see, the first few don't require you to do anything extraordinary, and can be done even if you've only been programming for two weeks! [See this relevant XKCD](https://xkcd.com/519/).
+
+> You might have heard that the tech industry changes quickly, trends come and go and technologies become obselete quickly. In addition, many companies have proprietary tools internal to the company. That's why it doesn't really matter *what* you learn, as long as you learn *something*. That signals to employers that you can learn whatever is necessary on the job.
+
+### Becoming and staying motivated
+
+It's ideal that you only do side projects if you're enjoying it. That way, it's sustainable, and just a generally healthy thing to do.
+
+But realistically, expect that it can take a few tries to enjoy something. Not all aspects of programming appeal to everyone. It's possible that the stuff you learned in class so far is pretty dry. Some people will enjoy UI/visual/graphics programming. Some people like math and algorithms. Some people like systems. Some people like building products. Some people like to do social good via tech. If you've committed to programming in some way anyway (e.g. enrolled in a CS degree), might as well do some exploration to see what you really like.
```

Comment:
I think something relevant to this section is that it's totally okay to abandon a project and work on something you think will be more interesting - do you wanna add a little bit about that here?